addons/website_sale/views/templates.xml

```diff
@@ -993,6 +993,8 @@
                                 </div>
                             </form>
                             <p t-elif="not product.active" class="alert alert-warning">This product is no longer available.</p>
+                            <p t-elif="product.recurring_invoice and (request.session['website_sale_cart_quantity'] if 'website_sale_cart_quantity' in request.session else website.sale_get_order().cart_quantity or 0) > 0" class="alert alert-warning">
```

Comment:
`recurring_invoice` is created in `sale_subscription` so you can't use it in `website_sale`. Please do this in `website_sale_subscription`